app/js/components/labOrderEntry/PastOrders.jsx

```diff
@@ -14,9 +15,9 @@ const PastOrders = ({
       </thead>
       <tbody>
         {orders.map(order => (
-          <tr key={order.id}>
-            <td>{order.date}</td>
-            <td>{order.details}</td>
+          <tr key={order.uuid}>
+            <td>{moment(order.dateActivated).format('DD MMM YYYY')}</td>
+            <td>{order.concept.display}</td>
```

Comment:
I believe `Moment` is a pretty heavy library and if this is all we need it for in this codebase presently: 
```
<td>{moment(order.dateActivated).format('DD MMM YYYY')}</td>
```
I suggest you write a simple javascript utility to do the date formatting.

Reply:
We would need it later for much more things other than this, so it's better we start using it than write a function now and then it'll become redundant later on.

Reply:
I think it's definitely makes sense to use a Javascript date library as we will have many occasions to use it across the app.  And we have used moment a lot in the past... however, we are already including date-fns in the application, I believe. Is there a method we could use from there instead?